package_bag/routines.py

```diff
@@ -43,10 +49,71 @@ def list_to_download(self):
         for filename in files_in_bucket:
             if not expected_file_name(filename):
                 files_in_bucket.remove(filename)
-            elif Bag.objects.filter(original_bag_name=join(self.src_dir, filename)):
+            elif Bag.objects.filter(original_bag_name__contains=filename):
                 files_in_bucket.remove(filename)
```

Comment:
I understand that concern, but given the volume of bags it would be very annoying to delete them manually.

One solution I can think of is an update like what happens in the Aurora workflow when an ingest is complete, HOWEVER in this case we would need to have success in Pictor and Aquarius in order to successfully say everything is good. Another solution is to delete the bag in S3 after delivering to Ursa Major, which means that the bag would have at least gotten through Zorya successfully.

Reply:
It looks like bags are currently deleted from S3 after they are downloaded, which I think is fine. What I'm concerned about are the removals in the S3Object finder - the only time those cases will crop up is when something has gone wrong (either a bag has a wrong file name or it failed and you're retrying with new data). In both of those cases, there are good reasons why you wouldn't want to just delete the bag.

Reply:
We're not deleting the bag here--we're just removing that bag from the list to download. If we don't, we'll redownload bags which I think would cause additional issues. Am I misunderstanding what you're saying?

Reply:
🤦 I am clearly ready for a vacation. Was misreading the `remove` as remove from S3 rather than a list. We're all good here.